Breakout/src/game_clock.cpp

```diff
@@ -0,0 +1,51 @@
+#include "game_clock.h"
+
+GameClock::GameClock(float fps = 60.f) : m_fps{ fps }
```

Comment:
Placing default values of a method's parameter in the header file makes it easier  for someone who uses this class to know, which parameters actually need to be passed. Next to that it tells the C++ compiler that this parameter is optional. Otherwise you get an error when only including the header file of this class somewhere.
```suggestion
GameClock::GameClock(float fps) : m_fps{ fps }
```
Please initialize the rest of the member attributes here as well (`m_elapsed` and `m_clock`)